docs/concepts/workloads/vm.md

```diff
@@ -84,15 +84,45 @@ It is possible to update parts of an existing `VirtualMachine` resource. Some fi
 
 ## Resources
 
-Some of a VM's hardware resources are derived, and there are some that may be influenced directly by a user.
-
-// TODO ([github.com/vmware-tanzu/vm-operator#105](https://github.com/vmware-tanzu/vm-operator/issues/105))
+Some of a VM's hardware resources are derived from the policies defined by your infrastructure administrator, others may be influenced directly by a user.
 
 ### CPU and Memory
+CPU and memory of a VM are derived form the `VirtualMachineClass` that is used to create the VM. Specifically, the `memoryMB` and `numCPUs` properties in the `configSpec` field of the `VirtualMachineClass` resource dictate the exact number of virtual CPUs, and the virtual memory (in MegaBytes) that the VM will be created with.
```

Comment:
What do you think moving `(in MegaBytes)` to line 117 in `!!! note "Units for CPU reservations/limits are in MHz"`? So the units are in the same place and reader doesn't get distracted here.

Reply:
Yeah, makes sense.  Done.